app/controllers/reports/regions_controller.rb

```diff
@@ -161,6 +161,9 @@ def find_region
     report_scope = report_params[:report_scope]
     @region ||= authorize {
       case report_scope
+      when "organization"
```

Comment:
In the specs, I see a series of tests like, "retrieves block data". This should probably have some similar specs.